hw_diag/tests/test_views_diagnostics.py

```diff
@@ -71,7 +71,11 @@ def test_initFile_output(self):
 
     @patch.dict(
         os.environ,
-        {'FIRMWARE_VERSION': '1337.13.37', 'DIAGNOSTICS_VERSION': 'aabbffe'}
+        {
```

Comment:
Same as above, we don't need this in `/initFile.txt` payload.

Reply:
@Kashif please correct me if I'm wrong. This line is not about testing the initFile.txt, it is for `/version` and we must report amd test it under `/version`. How can we run `test_version_endpoint()` without setting the environment var?

Reply:
kindly refrain from randomly adding random usernames to your issue... I believe this is a typo?

Reply:
> e must report amd test it under `/version`. How can we run `test_version_endpoint()` without setting the environment var?

Yes my bad, this is fine for version endpoint. And without addressing "Kashif Rasool", yes it was a typo.